site/content/docs/4.3/getting-started/theming.md

```diff
@@ -370,15 +370,19 @@ Here are the variables we include (note that the `:root` is required). They're l
 
 {{< highlight css >}}
 {{< root.inline >}}
+{{- $variables := readFile "scss/_variables.scss" -}}
+{{- $fontSansSerif := findRE "(\\$font-family-sans-serif:\\s+)(.*)(!default)" $variables -}}
```

Comment:
@bep: do you have any suggestions how one can get a capturing group from `findRE`?